src/pages/new-chat/new-chat.ts

```diff
@@ -41,28 +52,59 @@ export class NewChatComponent implements OnInit {
     });
   }
 
-  private findUsers(): ObservableCursor<User> {
+  findContacts(): Promise<Contact[]> {
+    // If we're running this in the browser, don't look for any contacts
+    if (!navigator.hasOwnProperty('contacts')) return Promise.resolve();
+
+    const fields = <ContactFieldType[]>['phoneNumbers'];
+
+    // Look for all the available phone numbers in contacts
+    return Contacts.find(fields, {
+      hasPhoneNumber: true,
+      multiple: true,
+      desiredFields: fields
+    });
+  }
+
+  subscribeUsers(contacts?: any[]): Subscription {
+    // Fetch all users in contacts list. If no contacts list was provided, will
+    // fetch all available users in database. This behavior is **not** recommended
+    // in a production application since it will probably overload both client
+    // and server
+    return MeteorObservable.subscribe('users', contacts).subscribe(() => {
```

Comment:
See my example with the `transformedContacts` - this way you will pass `number[]` to here